capstone-project/src/main/angular-webapp/src/app/info-window/info-window.component.ts

```diff
@@ -31,7 +31,8 @@ export class InfoWindowComponent implements OnInit {
     this.animal = animalValue;
     this.description = descriptionValue;
     this.reporter = reporterValue;
-    this.submitEvent.emit({animal: animalValue, description: descriptionValue, reporter: reporterValue})
+    const idToken = (this.user ? this.user.idToken : "")
+    this.submitEvent.emit({animal: animalValue, description: descriptionValue, reporter: reporterValue, userToken: idToken})
```

Comment:
I don't think it makes sense for the info window component to pass the user ID to the map component.

That's why you have the UserService, just inject it in the map component